mitodl/mit-open
# [NextJS] configure query client like on `main`

### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/5543

### Description (What does it do?)
This PR adds most of the query config from `main` branch to the NextJS app.

### How can this be tested?
1. Load the homepage with network tab open. API requests should be made once, not over and over.
2. When authenticated, featured courses on homepage should have correct filled-in / unfilled bookmark icons.
